legal-api/src/legal_api/resources/v2/business/business_documents.py

```diff
@@ -76,41 +77,64 @@ def _get_document_list(business):
 
     if business.legal_type == Business.LegalTypes.COOP.value:
-        coop_documents = _get_coop_documents_list(business)
-        for coop_doc_key, coop_doc_value in coop_documents.items():
-            documents['documents'][coop_doc_key] = coop_doc_value
+        documents['documentsInfo'] = {}
+        coop_documents, coop_info = _get_coop_documents_and_info(business)
+        for k, v in coop_documents.items():
+            documents['documents'][k] = v
+        for k, v in coop_info.items():
+            documents['documentsInfo'][k] = v
 
     return jsonify(documents), HTTPStatus.OK
 
 
-def _get_coop_documents_list(business):
-    """Get certified memorandum and rules for coop."""
-    coop_documents = {}
+# This is used as part of entity snapshot in business-edit-ui.
+def _get_coop_documents_and_info(business):
+    """Get certified memorandum and rules documents + info for coop."""
+    documents, info = {}, {}
 
     if not business:
-        return coop_documents
+        return documents, info
 
     base_url = current_app.config.get('LEGAL_API_BASE_URL')
     base_url = base_url[:base_url.find('/api')]
-    business_id = business.id
-    business_identifier = business.identifier
-
-    coop_rules_document = Document.find_by_business_id_and_type(business_id, DocumentType.COOP_RULES.value)
-
-    if coop_rules_document:
-        coop_rules_filing = Filing.find_by_id(coop_rules_document.filing_id)
-        coop_rules_doc_url = url_for('API2.get_documents', **{'identifier': business_identifier,
-                                                              'filing_id': coop_rules_filing.id,
+    business_id, business_identifier = business.id, business.identifier
+
+    info['certifiedRules'], info['certifiedMemorandum'] = {}, {}
+
+    sr_filing = Filing.get_filings_by_types(business_id, ['specialResolution'])
+    if rules_document := Document.find_by_business_id_and_type(business_id, DocumentType.COOP_RULES.value):
+        rules_filing = Filing.find_by_id(rules_document.filing_id)
+        rules_doc_url = url_for('API2.get_documents', **{'identifier': business_identifier,
+                                                         'filing_id': rules_filing.id,
+                                                         'legal_filing_name': None})
+        documents['certifiedRules'] = f'{base_url}{rules_doc_url}/certifiedRules'
+        filing_date_str = LegislationDatetime.format_as_legislation_date(rules_filing.filing_date.isoformat())
+        file_name = f'{business.identifier} - Certified Rules - {filing_date_str}.pdf'
+        info['certifiedRules'] = {
+            'key': rules_document.file_key,
+            'name': file_name,
+            'uploaded': rules_filing.filing_date.isoformat()
+        }
+    if sr_filing and sr_filing[0].filing_json['filing'].get('alteration', {}).get('rulesInResolution') is True:
+        info['certifiedRules']['includedInResolution'] = True
```

Comment:
We still want the original document carried along, we're just including the `includedInResolution`  and `updated` (a bit different than uploaded, uploaded is for the original document)